samples/sampler/stories/default/ArcSlider.js

```diff
@@ -0,0 +1,39 @@
+import {mergeComponentMetadata} from '@enact/storybook-utils';
+import {action} from '@enact/storybook-utils/addons/actions';
+import {select, number} from '@enact/storybook-utils/addons/knobs';
+import React from 'react';
+import {storiesOf} from '@storybook/react';
+
+import ArcSlider, {ArcSliderBase} from '@enact/agate/ArcSlider';
+
+ArcSlider.displayName = 'ArcSlider';
+const Config = mergeComponentMetadata('ArcSlider', ArcSliderBase, ArcSlider);
+
+// Set up some defaults for colors and radius
+const prop = {
+	colors: ['', '#000000', '#fdc902', '#986aad', '#0000ff'],
+	radius: [120, 150]
+};
+
+storiesOf('Agate', module)
+	.add(
+		'ArcSlider',
+		() => (
+			<ArcSlider
+				backgroundColor={select('backgroundColor', prop.colors, Config)}
+				endAngle={number('endAngle', Config, {range: true, min: 0, max: 360})}
+				foregroundColor={select('foregroundColor', prop.colors, Config)}
+				max={number('max', Config)}
+				min={number('min', Config)}
+				onMouseDown={action('onMouseDown')}
+				onTouchStart={action('onTouchStart')}
```

Comment:
Instead of `onMouseDown` and `onTouchStart`, please add the following props.

```suggestion
				onChange={action('onChange')}
				onDown={action('onDown')}
				onDrag={action('onDrag')}
				onDragEnd={action('onDragEnd')}
				onDragStart={action('onDragStart')}
```